### PR TITLE
Components: Add README for layout related components

### PR DESCRIPTION
**Pull request by renatoagds**
Start to close the gap in our components documentation.

This now introduce 3 new README files for:

- `useBreakpointMatch`
- `Container`
- `Col`

#### Changes proposed in this Pull Request:

* Add README.md

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Change base branch here in `Github` for mine.
* Navigate to `layout` folder and see all READMEs.

### CODE REVIEW

File: projects/js-packages/components/components/layout/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Layout
+
+This is our components and utilities to compose interfaces, it's based in our current specs for Layout composition and CSS Grid Layout.
```
Comment (retrofox) [inline projects/js-packages/components/components/layout/README.md]:
```suggestion
These are our components and utilities to compose interfaces, it's based on our current specs for Layout composition and CSS Grid Layout.
```

File: projects/js-packages/components/components/layout/col/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		<Text>World!</Text>
+	</Col>
+</Container>
+```
+
+## Props
+
+### className
+
+An custom class to append with the default ones.
```
Comment (retrofox) [inline projects/js-packages/components/components/layout/col/README.md]:
```suggestion
A custom class to append with the default ones.
```

File: projects/js-packages/components/components/layout/container/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		<Button>Click me!</Button>
+	</Col>
+</Container>
+```
+
+## Props
+
+### className
+
+An custom class to append with the default ones.
```
Comment (retrofox) [inline projects/js-packages/components/components/layout/container/README.md]:
```suggestion
A custom class to append with the default ones.
```

File: projects/js-packages/components/components/layout/container/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+```jsx
+// Space will be 16px
+<Container horizontalSpacing={ 2 } />
+```
+
+### horizontalGap
+
+The space between each row.
+
+Value is mutiplied per `8px`, following our specs.
```
Comment (retrofox) [inline projects/js-packages/components/components/layout/container/README.md]:
```suggestion
Value is multiplied per `8px`, following our specs.
```

File: projects/js-packages/components/components/layout/container/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+```jsx
+<Container fluid />
+```
+
+### horizontalSpacing
+
+The space on top and bottom of the container.
+
+Value is mutiplied per `8px`, following our specs.
```
Comment (retrofox) [inline projects/js-packages/components/components/layout/container/README.md]:
```suggestion
Value is multiplied per `8px`, following our specs.
```

File: projects/js-packages/components/components/layout/use-breakpoint-match/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- Required: `true`
+
+```javascript
+useBreakpointMatch('lg');
+useBreakpointMatch([ 'lg', 'sm' ]);
+```
+
+### operator
+
+An Single or Array of operator to match less, equal or greater than breakpoint.
```
Comment (retrofox) [inline projects/js-packages/components/components/layout/use-breakpoint-match/README.md]:
```suggestion
A Single or Array of operators to match less, equal, or greater than the breakpoint.
```

File: projects/js-packages/components/components/layout/use-breakpoint-match/README.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+In Array, it match against the same index from breakpoint.
```
Comment (retrofox) [inline projects/js-packages/components/components/layout/use-breakpoint-match/README.md]:
```suggestion
In Array, it matches against the same index from breakpoint.
```

File: projects/js-packages/components/components/layout/container/README.md (hunk 5 above)
Comment (oskosk) [reply projects/js-packages/components/components/layout/container/README.md]: Should be multiplied "by"

File: projects/js-packages/components/components/layout/container/README.md (hunk 4 above)
Comment (oskosk) [reply projects/js-packages/components/components/layout/container/README.md]: Should be multiplied "by"

**Comment by oskosk** [comment]
Uh sorry. I committed all of Damian's suggestions thinking this PR was read to review.

**Comment by renatoagds** [comment]
@oskosk It's ready for review

**Review by oskosk — APPROVED** [review]

**Comment by renatoagds** [comment]
r247826-wpcom

**Pull request merged**